pkg/capture/flow.go

```diff
@@ -103,12 +241,9 @@ func (f *FlowLog) transferAndAggregate() (newFlowMap map[string]*GPFlow, agg *ha
 	return
 }
 
-// EPHash is a typedef that allows us to replace the type of hash
-type EPHash [37]byte
-
 // GPFlow stores a goProbe flow
 type GPFlow struct {
```

Comment:
While we are at it. Should we now just call it `Flow` instead of `GPFlow`?

Reply:
Very good observation, absolutely!